### PR TITLE
ci: adds tag.yml

### PR DESCRIPTION
**Pull request by dana-rocha**

### CODE REVIEW

**Comment by dana-rocha** [comment]
Closes #1 and #3

File: .github/workflows/tag.yml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+          version_in_this_pr_patch=${BASH_REMATCH[3]}
+          version_in_this_pr_minor=${BASH_REMATCH[2]}
+          version_in_this_pr_major=${BASH_REMATCH[1]}
+          new_tag=$version_in_this_pr_major.$version_in_this_pr_minor.$version_in_this_pr_patch
+          echo "::set-output name=version::$new_tag"
+      - name: Bump tag
+        uses: mathieudutour/github-tag-action@v6.1
+        with:
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+          custom_tag: ${{ steps.getversion.outputs.version }}
```
Comment (jtyoung84) [inline .github/workflows/tag.yml:27]: Add a new line to the file

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
Add new line to end of file

File: .github/workflows/tag.yml (hunk 1 above)
Comment (dana-rocha) [reply .github/workflows/tag.yml]: Just added!

**Review by jtyoung84 — APPROVED** [review]

**Pull request merged**